packages/api/tests/lib/test-granule-delete.js

```diff
@@ -352,7 +313,7 @@ test.serial('deleteGranuleAndFiles() will not delete S3 Files if the PostgreSQL
   ]));
 });
 
-test.serial('deleteGranuleAndFiles() will not delete granule or S3 files if the Elasticsearch granule delete fails', async (t) => {
+test.serial('deleteGranuleAndFiles() will delete granule and S3 files', async (t) => {
```

Comment:
We already have this test. 
'deleteGranuleAndFiles() removes granules from PostgreSQL and files from PostgreSQL/S3